build.gradle

```diff
@@ -50,7 +50,6 @@ allprojects {
     }
 
     tasks.withType(org.jetbrains.intellij.tasks.RunIdeTask) {
-        systemProperty("aws.toolkits.enableTelemetry", false)
```

Comment:
might have missed the conversation, but are we leaving this on for testing now?

Reply:
Nah we are, 3 lines up we do it for all JavaExec so its redundant.